docs: no network request for performance strategy (#29288)
angular/angular: aio/content/guide/service-worker-config.md

The Angular service worker can use either of two caching strategies for data resources.

> * `performance`, the default, optimizes for responses that are as fast as possible. If a resource exists in the cache, the cached version is used, there will be no request from the network at all. This allows for some staleness, depending on the `maxAge`, in exchange for better performance. This is suitable for resources that don't change often; for example, user avatar images.

Reviewer: ```suggestion
* `performance`, the default, optimizes for responses that are as fast as possible. If a resource exists in the cache, the cached version is used, and no network request is made. This allows for some staleness, depending on the `maxAge`, in exchange for better performance. This is suitable for resources that don't change often; for example, user avatar images.
```
